nrfconnect/sdk-nrf
# samples: fmfu_smp_svr: Add to support APP serial DFU

Allow update the sample itself by mcuMgr based serial DFU.
Output logs to RTT, leaving UART_0 for mcuMgr to use.
Test steps:
 mcumgr conn add comX type="serial" connstring="COMX,baud=115200,mtu=512"
 mcumgr -c comX echo Hello
 mcumgr -c comX image upload build\zephyr\app_update.bin
 mcumgr -c comX image list
 mcumgr -c comX image test <slot_1 hash>
 mcumgr -c comX reset
 mcumgr -c comX image confirm <slot_0 hash>
 mcumgr -c comX image list
Refer to zephyr\samples\subsys\mgmt\mcumgr\smp_svr for more info.

Signed-off-by: Jun Qing Zou <jun.qing.zou@nordicsemi.no>
